locale/en/site.json

```diff
@@ -72,16 +84,18 @@
             "link": "docs/faq",
             "text": "FAQ"
         },
-        "api": {
-            "text": "API"
+        "api-lts": {
+            "link": "dist/latest-v4.x/docs/api",
```

Comment:
oh hmm, no that isn't ok. The docs are not translated to begin with. (due to various other issues)